src/reporters/apiary-reporter.coffee

```diff
@@ -177,38 +181,34 @@ class ApiaryReporter
     return data
 
   _performRequestAsync: (path, method, body, callback) =>
+    logger.debug("""\
+      Performing #{method.toUpperCase()} request to '#{@configuration.apiUrl}' + '#{path}', \
+      #{if body then 'with' else 'without'} body.\
+    """)
+
     buffer = ""
 
-    handleResponse = (res) =>
+    handleResponse = (res) ->
       res.setEncoding 'utf8'
 
-      res.on 'data', (chunk) =>
-        if @verbose
-          logger.log 'REST Reporter HTTPS Response chunk: ' + chunk
+      res.on 'data', (chunk) ->
+        logger.verbose('Apiary Reporter HTTP response chunk:', chunk)
         buffer = buffer + chunk
 
-      res.on 'error', (error) =>
-        if @verbose
-          logger.log 'REST Reporter HTTPS Response error.'
+      res.on 'error', (error) ->
+        logger.debug('Apiary Reporter HTTP response error:', error)
         return callback error, req, res
 
-      res.on 'end', =>
-        if @verbose
-          logger.log 'Rest Reporter Response ended'
+      res.on 'end', ->
+        logger.verbose('Apiary reporter Response received.')
 
         try
           parsedBody = JSON.parse buffer
         catch e
```

Comment:
Is caught error ignored intentionally? Wouldn't it help for faster orientation in JSON parse error?